src/EditorFeatures/TestUtilities2/Intellisense/TestState.vb

```diff
@@ -387,6 +387,17 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             Next
         End Sub
 
+        Public Sub AssertItemsInOrder(expectedOrder As (String, String)())
+            Dim session = GetExportedValue(Of IAsyncCompletionBroker)().GetSession(TextView)
+            Assert.NotNull(session)
+            Dim items = session.GetComputedItems(CancellationToken.None).Items
+            Assert.Equal(expectedOrder.Count, items.Count)
+            For i = 0 To expectedOrder.Count - 1
+                Assert.Equal(expectedOrder(i).Item1, items(i).DisplayText)
+                Assert.Equal(expectedOrder(i).Item2, items(i).Suffix)
+            Next
```

Comment:
Consider just building a second list of tuples, and then calling Assert.Equal() on the entire lists; that way if the ordering doesn't match you'll get the entire lists to compare rather than "A doesn't match B" and you have no idea where that went wrong.